lib/util/html.ts

```diff
@@ -1,7 +1,11 @@
-import { HTMLElement, parse as _parse } from 'node-html-parser';
+import { HTMLElement, Options, parse as _parse } from 'node-html-parser';
 
 export { HTMLElement };
 
-export function parse(html: string): HTMLElement {
+export function parse(html: string, config?: Partial<Options>): HTMLElement {
+  if (typeof config !== 'undefined') {
+    return _parse(html, config);
+  }
+
   return _parse(html);
```

Comment:
```suggestion
  return _parse(html, config);
```

simplify? Otherwise add a test to `./html.spec.ts` for coverage

Reply:
Added unit test.